src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
     }
```

Comment:
Consider creating a single instance of `Random` class and reuse it. Creating a new instance everytime we call the method can be inefficient.
Also, using `toString()` for enum is bad practice. It can lead to potential issues. Use `name()` instead.

![image](https://github.com/kacper-kukula/my-test-task/assets/105852806/76c5b16b-46a1-4035-9e3b-f86740b5765b)

![image](https://github.com/kacper-kukula/my-test-task/assets/105852806/61b3a6de-b3f9-46f7-bb32-2ec539e69fc4)